templates/register.twig

```diff
@@ -0,0 +1,204 @@
+<!--suppress ES6ModulesDependencies -->
+<script>
+    const Register = {
+        template: `
+        <div id="check-update">
+            <v-card-text>
+                <template v-if="'{{ error }}' === ''">
+                     <v-stepper v-model="slide">
+                        <v-stepper-header>
+                            <v-stepper-step :complete="slide > 1" step="1"> Welcome </v-stepper-step>
+                            <v-stepper-step :complete="slide > 2" step="2"> Terms of Service </v-stepper-step>
+                            <v-stepper-step :complete="slide > 3" step="3"> Profile </v-stepper-step>
+                            <v-stepper-step :complete="slide > 4" step="4"> Go! </v-stepper-step>
+                        </v-stepper-header>
+                    </v-stepper>
+                    <v-divider style="margin: 15px 0 15px 0"/>
+                    <v-form>
+                        <template v-if="slide === 1">
+                            <h2>Welcome, {{ username }}!</h2>
+                            <br>
+                            <p>
+                                We need to check few things before we can continue.
+                                Some are standard things, some are customizations for your account. <br> <br>
+                                <i>Thank you for using GameTactic services!</i>
+                            </p>
+                        </template>
+                        <template v-if="slide === 2">
+                            <p>
+                                Please click and read <a target="_blank" href="https://gametactic.eu/tos"> terms of service </a> fully.
+                                When you're done, please agree them bellow.
+                                <v-checkbox v-model="tos" @click="cannotContinue = false" :disabled="tos" label="I Accept Terms of Service" />
+                            </p>
+                        </template>
+                        <template v-if="slide === 3">
+                            <v-form v-model="usernameValid">
+                                <p> Please select username. It will be same, even you login via different service. </p>
+                                <v-text-field v-model="username" :rules="nameRules" label="Username" :counter="32"
+                                    placeholder="{{ username }}" @input="onProfileFields()" required />
+                                <v-select v-model="region" :items="regions" label="Region" item-value="val" item-text="key" @change="onProfileFields()"/>
+                            </v-form>
+                        </template>
+                        <template v-if="slide === 4">
+                            <h2>Thank you {{ '{{ username }}' }}!</h2>
+                            <br>
+                            <p>
+                                You are now ready to go. Just press "continue" and you will be redirected
+                                back where you left off.
```

Comment:
You are now ready to go. Just press "continue" and you will be redirected back to where you came from*